config/locales/support_interface/reference_status.yml

```diff
@@ -8,3 +8,5 @@ en:
       feedback_provided: Feedback provided
       feedback_refused: Reference declined
       email_bounced: Email bounced
+    confidential_warning: Confidential do not share with the candidate
```

Comment:
```suggestion
    confidential_warning: Confidential, do not share with the candidate
```

Reply:
Thanks - i've run this past our content designer. 

We've decided to go with a colon instead.